### PR TITLE
[STREAMING][DOCS] Fixes and code improvements for checkpointing

### PR DESCRIPTION
**Pull request by jaceklaskowski**

### CODE REVIEW

**Comment by jaceklaskowski** [comment]
@srowen Mind having a look? I'd appreciate. Thanks!

File: docs/streaming-programming-guide.md [hunk 1]
```diff
@@ -1787,11 +1788,11 @@ reliable file system (e.g., HDFS, S3, etc.) to which the checkpoint information
 This is done by using `streamingContext.checkpoint(checkpointDirectory)`. This will allow you to
 use the aforementioned stateful transformations. Additionally,
 if you want to make the application recover from driver failures, you should rewrite your
-streaming application to have the following behavior.
+streaming application to have the following behavior:
```
Comment (srowen) [inline docs/streaming-programming-guide.md]: A couple of the trivial formatting changes I agree with; most of the changes are not improvements (e.g. "being" is slightly more accurate below). I would close this PR

**Comment by HyukjinKwon** [comment]
ping @jaceklaskowski

**Comment by srowen** [comment]
I think this should just be closed.

**Pull request closed without merging**